manual/source/Tutorial/Algorithms/PatternMatching/IndexedPatternMatching.rst

```diff
@@ -17,34 +17,21 @@ Duration
   30 min
 
 Prerequisites
-  :ref:`tutorial-datastructures-sequences`, :ref:`tutorial-datastructures-indices`
+  :ref:`tutorial-datastructures-sequences`, :ref:`tutorial-datastructures-indices`, :ref:`tutorial-algorithms-pattern-matching-online`
 
 Overview
 --------
 
-SeqAn provides two methods for searching for a pattern in index structures.
-One method uses iterators and is similar to traversing search trees or tries.
-The tutorial :ref:`tutorial-datastructures-indices-index-iterators` explains this method in more detail.
-In this section you will learn how to find a pattern with the :dox:`Finder` interface.
+The :dox:`Finder` is an object that stores all necessary information for searching for a pattern. We have learned how to use it in :ref:`tutorial-algorithms-pattern-matching-online` tutorial.
 
-The :dox:`Finder` is an object that stores all necessary information for searching for a pattern using an index.
-The following line of code shows how the :dox:`Finder` is initialized.
+The following line of code shows how the :dox:`Finder` is initialized with an index. In this example, we search for the pattern ``ACGT``.
 
 .. includefrags:: demos/tutorial/indices/base.cpp
       :fragment: finder
 
-After initialization it is possible to use the :dox:`Finder#find` function in order to trigger a search for all occurrences of a given pattern in the underlying :dox:`String` or :dox:`StringSet`.
-In this example, we search for the pattern ``ACGT``:
-
-.. includefrags:: demos/tutorial/indices/base.cpp
-      :fragment: finder2
-
 Calling the function :dox:`Finder#find` invokes the localization of all occurrences of a given pattern.
 It works by modifying pointers of the ``Finder`` to tables of the index.
-For example, the :dox:`Finder` of ``esaIndex`` stores two pointers, pointing to the first and last suffix array entry that stores an occurrence of the pattern.
-
-The return value of the :dox:`Finder#find` function tells us whether or not a given pattern occurs in the text.
-Furthermore, if there are several instances of a pattern, consecutive calls of :dox:`Finder#find` will modify the :dox:`Finder` such that it points to the next occurrence after each call:
```

Comment:
I couldn't find this explanation neither in the related ``online matching tutorial`` nor in the other required tutorials (in a quick way). Sorry if I have missed it. Otherwise I would prefer to keep this sentence :smile: